themes/default/views/editor/objects/summary_html.php

```diff
@@ -40,8 +40,8 @@
 ?>
 		<div id="toggleCollapsedButton">
 			<a href="#">
-				<?php print caNavButton($this->request, __CA_NAV_BUTTON_COLLAPSE__); ?>
-				<?php print caNavButton($this->request, __CA_NAV_BUTTON_EXPAND__); ?>
+				<span class="collapse"><?php print caNavIcon($this->request, __CA_NAV_BUTTON_COLLAPSE__); ?></span>
```

Comment:
Added the class names here, so they are in the same file as the JavaScript that references them.  Change `caNavButton` to `caNavIcon` as discussed.